homeassistant/components/deconz/sensor.py

```diff
@@ -108,6 +111,17 @@ class DeconzSensorDescription(Generic[T], SensorEntityDescription):
 
 
 ENTITY_DESCRIPTIONS: tuple[DeconzSensorDescription, ...] = (
+    DeconzSensorDescription[AirPurifier](
+        key="air_purifier_filter_run_time",
+        supported_fn=lambda device: True,
+        update_key="filterruntime",
+        name_suffix="Filter Time",
+        value_fn=lambda device: device.filter_run_time,
+        instance_check=AirPurifier,
+        device_class=SensorDeviceClass.DURATION,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        native_unit_of_measurement=UnitOfTime.SECONDS,
```

Comment:
Should we add suggested_unit_of_measurement? A sensor in seconds doesn't sound pleasant

Reply:
Done